cfbs.json

```
@@ -139,6 +139,52 @@
       "subdirectory": "management/every-minute",
       "steps": ["json def.json def.json"]
     },
+    "file-permissions": {
+      "description": "Manages file permissions",
+      "tags": ["management"],
+      "repo": "https://github.com/nickanderson/cfengine-file-permissions",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "a678160011b834557f1ab42a081ea84adf8622df",
+      "steps": [
+        "copy file-permissions.cf services/file-permissions.cf",
```

I'd suggest moving the file to:
```suggestion
        "copy file-permissions.cf services/cfbs/modules/file-permissions/file-permissions.cf",
```